frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  getVmStatus,
+  VM_STATUS_IMPORTING,
+  VM_STATUS_V2V_CONVERSION_IN_PROGRESS,
+} from 'kubevirt-web-ui-components';
+import { V1Pod } from 'openshift-typescript-api/esm';
+import { V1VirtualMachineInstanceMigration } from 'kubevirt-typescript-api/esm';
+
+import { Kebab } from '@console/internal/components/utils';
+import { isVmRunning } from '../../selectors/vm';
+import { startStopVmModal } from '../modals/start-stop-vm-modal';
+
+type ActionArgs = {
+  pods: V1Pod[];
+  migrations: V1VirtualMachineInstanceMigration[];
+};
+
+const isImporting = (vm, { pods, migrations }: ActionArgs) => {
+  const status = getVmStatus(vm, pods, migrations);
+  return (
+    status && [VM_STATUS_IMPORTING, VM_STATUS_V2V_CONVERSION_IN_PROGRESS].includes(status.status)
+  );
+};
+
+const menuActionStart = (kind, vm, actionArgs) => {
```

Comment:
You'll want to add RBAC checks to these actions. For example,

https://github.com/openshift/console/blob/master/frontend/public/components/utils/kebab.tsx#L64-L78